packages/sdk/src/reliable_channel/index.ts

```diff
@@ -0,0 +1,633 @@
+import { TypedEventEmitter } from "@libp2p/interface";
```

Comment:
can you split this file into `exports` and `implementation`
example - https://github.com/waku-org/js-waku/blob/67a7287931a8feba01539106b2457f03e86c3b3b/packages/sdk/src/peer_manager/index.ts

I'll check if it possible to make ESLint rule for that

Reply:
I think I found ESLint rule for that https://github.com/waku-org/js-waku/pull/2611